esmvaltool/utils/cmorizers/obs/cmorize_obs_Landschutzer2014.py

```diff
@@ -0,0 +1,117 @@
+"""
+# #############################################################################
+# ESMValTool CMORizer for WOA data
+# #############################################################################
+#
+# Tier
+#    Tier 2: other freely-available dataset.
+#
+# Source
+#    ftp://ftp.nodc.noaa.gov/nodc/archive/arc0105/0160558/1.1/data/0-data/
+#
+# Last access
+#    20190227
+#
+# Download and processing instructions
+#    Download the following file in ${RAWOBS}/Tier2/Landschutzer2014:
+#     pco2_1998-2011_ETH_SOM-FFN_CDIAC_G05.nc
+#      
+#
+# Modification history
+#    20190227-A_lova_to: written.
+#
+# #############################################################################
+"""
+
+import datetime
+import logging
+import os
+import glob
+
+import iris
+import netCDF4 as nc
+from cf_units import Unit
+import numpy as np
+
+from .utilities import (_add_metadata,
+                        _convert_timeunits,
+                        _fix_coords,
+                        _read_cmor_config,
+                        _roll_cube_data,
+                        _save_variable)
+
+logger = logging.getLogger(__name__)
+
+# read in CMOR configuration
+cfg = _read_cmor_config('Landschutzer2014.yml')
+proj = cfg['proj']
+timestamp = datetime.datetime.utcnow()
+timestamp_format = "%Y-%m-%d %H:%M:%S"
+now_time = timestamp.strftime(timestamp_format)
+proj['metadata_attributes']['CMORcreated'] = now_time
+VAR_TO_CMOR = cfg['VAR_TO_CMOR']
+VAR_TO_FILENAME = cfg['VAR_TO_FILENAME']
+STANDARD_NAMES = cfg['STANDARD_NAMES']
+LONG_NAMES = cfg['LONG_NAMES']
+NC4_ZIP = cfg['NC4_ZIP']
+# 
+ALLVARS=list(VAR_TO_CMOR.keys())
+
+
+def _fix_metadata(cube, var):
+    """Fix all aspects of metadata for different vars."""
+    logger.info("Fixing units...")
+    if var in ['fgco2',] :
+        cube.units = Unit('kg m-2 s-1')
+    if var in ['spco2','dpco2' ] :
+        cube.units = Unit('Pa')
+    return cube
+
+
+def _fix_data(cube, var):
+    """Specific data fixes for different variables."""
+    logger.info("Fixing data ...")
+    # fix for missing value definition 
+    cube.data=np.ma.masked_values(cube.data, cube.data.fill_value)
+    if var in ['fgco2'] :
+        """ Assume standard year 365_day"""
+        cube.data = cube.data * -12.01 / 1000. / (86400. * 365.)   # Reverse flux sign & Convert from mol m^{-2} yr^{-1} to kg m-2 s-1
```

Comment:
why are you multiplying the cube data by years?

Reply:
I actually convert the units from years to seconds. I admit it is not very elegant in this way ... any suggestion?

Reply:
you need to convert the time units to cmor standard units eg `days since 1950-1-1 00:00:00` and this is reflected in `utilities.py`: first check is if there are years 0000 that are not supported at load point - this is done in `utilties._convert_timeunits()`; then in `utilities._fix_coords()` there is a function that converts your time to a cmor standard time axis, no need to do manual operations - also not on the var data itself because you are altering the actual values of the data

Reply:
I guess we may messed up the time issue...
Here we need to scale the field values from the unit  mol m^-2 **yr^-1** to kg m-2 **s-1**
and not to alter the time dimension.
As fgco2 is a flux its unit includes also the time!